server/api/handlers/account_info.go

```diff
@@ -2,15 +2,37 @@ package handlers
 
 import (
 	"context"
+	"github.com/eoscanada/eos-go"
+	"platform-backend/models"
 	"platform-backend/server/api/ws_interface"
 )
 
+type PlayerInfoResponse struct {
+	Balance          eos.Asset         `json:"balance"`
+	ActivePermission eos.Authority     `json:"activePermission"`
+	OwnerPermission  eos.Authority     `json:"ownerPermission"`
+	LinkedCasinos    []*CasinoResponse `json:"linkedCasinos"`
+}
+
+func toPlayerInfoResponse(p *models.PlayerInfo) *PlayerInfoResponse {
```

Comment:
I would do so. in my opinion so beautiful
``` func (p *models.PlayerInfo) toPlayerInfoResponse() *PlayerInfoResponse```
```return player.toPlayerInfoResponse(), nil```

Reply:
I cant specify method for external type. In that case `models.PlayerInfo` is external.
It's possible only if we would define that methods in models package, but then we get abstraction leak.